system-addon/lib/TopStoriesFeed.jsm

```diff
@@ -224,36 +233,25 @@ this.TopStoriesFeed = class TopStoriesFeed {
   }
 
   // If personalization is turned on we have to rotate stories on the client.
-  // An item can only be on top for two iterations (1hr) before it gets moved
-  // to the end. This will later be improved based on interactions/impressions.
+  // Items only be on top for the time configured in settings.
+  // Once an item is expired it will be sorted to the back.
```

Comment:
nit: let's rephrase this comment a bit. Perhaps "If personalization is turned on, we have to rotate all the stores on the client so that the active stories are relocated to the front of the list, followed by those expired ones."?